content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +34,184 @@ We have deprecated the following cert-manager APIs:
 - `acme.cert-manager.io/v1alpha3`
 - `acme.cert-manager.io/v1beta1`
 
-These APIs will be removed in cert-manager `v1.6.0`.
-If you have a cert-manager installation that is using or has previously used these deprecated APIs you may need to upgrade your cert-manager custom resources and CRDs. This needs to be done before upgrading to cert-manager `v1.6.0`.
-See cert-manager [docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) for more detailed upgrade instructions.
+These APIs will be removed in cert-manager 1.6.
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you have a cert-manager installation that is using or has previously used
+these deprecated APIs, you may need to upgrade your cert-manager custom
+resources and CRDs.
+
+This needs to be done before upgrading to cert-manager 1.6. See cert-manager
+[docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources)
+for more detailed upgrade instructions.
+
+{{% /pageinfo %}}
+
+### Helm chart: `securityContext` defaults to non-root
+
+The Helm chart now follows the current Pod hardening best practices as defined
+by the Kyverno [`pod-security
+restricted`](https://kyverno.io/policies/pod-security/#restricted) policy.
+
+To pass the validation, the controller, webhook, and cainjector Pods are now
+running as non-root:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    runAsNonRoot: true
+```
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you are using custom containers that run as root with the Helm chart, you
+will need to set this back to `false`.
+
+{{% /pageinfo %}}
+
+### CA, Vault and Venafi issuer handling of `ca.crt` and `tls.crt`
+
+The CA, Vault, and Venafi issuer now produce a `tls.crt` that is de-duplicated,
+in the correct order (leaf at the top, issuing certificate at the bottom) and
+verified (i.e. each signature can be verified).
+
+The CA issuer now produces a `ca.crt` that contains the root-most CA instead of
+the intermediate CA when the issuing certificate is an intermediate CA. By
+root-most, we mean that `ca.crt` is the root of the chain of certificates that
+cert-manager knows about, but it might not be the self-signed root CA.
+
+[#3982]: https://github.com/jetstack/cert-manager/pull/3982 "All issuers + Vault issuer"
+[#3983]: https://github.com/jetstack/cert-manager/pull/3983 "Venafi issuer"
+[#3985]: https://github.com/jetstack/cert-manager/pull/3985 "CA issuer"
+
+{{% pageinfo color="warning" %}}
+
+⛔️  You may need to adjust systems that consume the `ca.crt` from Secrets
+managed by cert-manager with the CA issuer.
+
+{{% /pageinfo %}}
```

Comment:
@erikgb @JoshVanL @wallrj does this paragraph make sense?

Context: [this Slack thread](https://kubernetes.slack.com/archives/CDEQJ0Q8M/p1623770041428700)

Reply:
Makes sense to me :+1:

Reply:
Makes sense to me too, but this is a definately bit "complex". The `ca.crt` will contain the "most" root CA it is aware of. 😄

Reply:
I mean: The self-signed CA, aka root-CA is not really required to issue certificates.

Reply:
Thanks Erik!! Your phrasing is clearer. I went with:

> The CA issuer now produces a `ca.crt` that contains the "most" root CA that cert-manager is aware of. `ca.crt` may thus not be the actual self-signed root CA, since cert-manager may not be aware of it.